NitroxClient/GameLogic/InitialSync/QuickSlotInitialSyncProcessor.cs

```diff
@@ -0,0 +1,57 @@
+using System.Collections;
+using System.Collections.Generic;
+using NitroxClient.GameLogic.InitialSync.Base;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.DataStructures;
+using NitroxModel.Packets;
+
+namespace NitroxClient.GameLogic.InitialSync;
+
+public class QuickSlotInitialSyncProcessor : InitialSyncProcessor
+{
+    public QuickSlotInitialSyncProcessor()
+    {
+        DependentProcessors.Add(typeof(PlayerInitialSyncProcessor));  // the player needs to be configured before we can set quick slots.
+        DependentProcessors.Add(typeof(EquippedItemInitialSyncProcessor)); // we need to have the items spawned into our inventory before we can quick slot them.
+    }
+
+    public override IEnumerator Process(InitialPlayerSync packet, WaitScreen.ManualWaitItem waitScreenItem)
+    {
+        int nonEmptySlots = 0;
```

Comment:
This isn't used

Reply:
It is used below at the final log line.

Reply:
Well yeah but it is not modificated after being initialized